src/signals/incident-management/containers/IncidentSplitContainer/components/SplitForm/index.js

```diff
@@ -82,6 +82,8 @@ const SplitForm = ({ incident, attachments, onHandleCancel, onHandleSubmit }) =>
   const [isVisible, setVisibility] = useState(false);
 
   const handleSubmit = useCallback(() => {
+    if (!incident) return;
```

Comment:
Can you add a test case for this condition?

Reply:
done!